README.md

```diff
@@ -116,30 +149,3 @@ To set up the project, navigate to the project folder in terminal, then run `npm
 * Add the [react dev tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) extension to chrome, and switch to the "React" tab after pressing Ctrl+Shift+J. You can see the react component tree. Once you click on a component in the tree, you will have access to it in the console as the variable `$r`. For example, you can check the props or state using `$r.props` or `$r.state`.
 * If you think a previous commit broke your feature, use [git's builtin debugging tools](https://git-scm.com/book/en/v2/Git-Tools-Debugging-with-Git)
 * If you fix a bug, **write a test for it**.
-
-## Creating a user ##
```

Comment:
I'm hesitant to completely remove this section, since it's the only part that explains how to use the meteor shell. Maybe just change the title and opening line to:

## Working wit the database ##

You can create users and posts using the LessWrong interface itself, but you may occasionally want to create objects directly via the database for testing purposes. Some examples:

If you wanted to create a new user... 

(should be sufficient and leave the rest as-is)

Reply:
Thanks for reviewing! I'll make this change hopefully within the next week or so.